### PR TITLE
Suggest changes - initial frontend

### PR DESCRIPTION
**Pull request by Tymek**
## About the changes
Setting up a page and refactoring layout for suggest<>review flow.

Relates to roadmap item: 4-eyes approval

### CODE REVIEW

File: src/server-dev.ts [hunk 1]
```diff
@@ -40,6 +40,7 @@ process.nextTick(async () => {
                         responseTimeWithAppName: true,
                         personalAccessTokens: true,
                         syncSSOGroups: true,
+                        suggestChanges: true,
```
Comment (sjaanus) [inline src/server-dev.ts:43]: I guess you need to update snapshot for this.

File: frontend/src/component/layout/LayoutPicker/LayoutPicker.tsx [hunk 2]
```diff
@@ -1,37 +1,19 @@
...
-    '/forgotten-password',
-    '/splash/:splashId',
-    '/404',
-];
+export const LayoutPicker: FC<ILayoutPickerProps> = ({
+    isStandalone,
+    children,
+}) => (
+    <ConditionallyRender
+        condition={isStandalone === true}
```
Comment (sjaanus) [inline frontend/src/component/layout/LayoutPicker/LayoutPicker.tsx:15]: Does this need `===` or it can be used straight away?

**Review by sjaanus — APPROVED** [review]
Other than small fix, LGTM!

File: frontend/src/component/layout/LayoutPicker/LayoutPicker.tsx (hunk 2 above)
Comment (Tymek) [reply frontend/src/component/layout/LayoutPicker/LayoutPicker.tsx:15]: can be `Boolean(isStandalone)`, because prop can be `undefined`

**Pull request merged**